c10/core/Device.cpp

```diff
@@ -6,26 +6,10 @@
 #include <array>
 #include <exception>
```

Comment:
don't you need to add #include <ctype.h> since you use isalpha?

Reply:
Ah! that's an oversight - thanks!

Reply:
Strangely it worked w/o the header - probably because of transitive includes, but better to include it directly.